src/components/TokenBottomSheet.test.tsx

```diff
@@ -80,34 +88,113 @@ const onTokenSelectedMock = jest.fn()
 const onCloseMock = jest.fn()
 
 describe('TokenBottomSheet', () => {
-  beforeAll(() => {
-    // @ts-ignore This avoids an error, see: https://github.com/software-mansion/react-native-reanimated/issues/1380
-    global.__reanimatedWorkletInit = jest.fn()
-  })
-
   beforeEach(() => {
+    jest.useFakeTimers()
     jest.clearAllMocks()
   })
 
-  function renderPicker(visible: boolean, isInvite: boolean = false) {
+  afterEach(() => {
+    jest.useRealTimers()
+  })
+
+  function renderPicker(visible: boolean, searchEnabled: boolean = false) {
     return render(
       <Provider store={mockStore}>
         <TokenBottomSheet
+          title="testTitle"
           isVisible={visible}
           origin={TokenPickerOrigin.Send}
           onTokenSelected={onTokenSelectedMock}
           onClose={onCloseMock}
           tokens={tokens}
+          searchEnabled={searchEnabled}
         />
       </Provider>
     )
   }
 
+  // TODO before merge: Investigate why this tests are failing if put at the end of the file.
+  it('renders and behaves correctly when the search is enabled', async () => {
+    jest.useRealTimers()
```

Comment:
Can we avoid this by advancing timers?
Otherwise it slows down tests unnecessarily.

Reply:
I have tried to use fake timers, but I couldn't make it work. I'll give it a try today again, but I think I will need help. Can I schedule a short meeting to do pair programming session to solve this?

Reply:
Sorry didn't see that. Feel free to schedule meetings whenever you want.

Reply:
No worries, I finally found the problem was solved by using `jest.useFakeTimers('modern')` 🙌